renku/core/commands/login.py

```diff
@@ -80,14 +100,40 @@ def _get_url(parsed_endpoint, path, **query_args):
     return parsed_endpoint._replace(path=path, query=query).geturl()
 
 
-def _store_token(client, parsed_endpoint, token):
-    client.set_value(section=CONFIG_SECTION, key=parsed_endpoint.netloc, value=token, global_only=True)
+def _store_token(client, parsed_endpoint, access_token):
+    client.set_value(section=CONFIG_SECTION, key=parsed_endpoint.netloc, value=access_token, global_only=True)
     os.chmod(client.global_config_path, 0o600)
 
 
+def _store_git_token(client, access_token):
+    # NOTE: Do not use "Authorization: Bearer token" because it interferes with Git LFS authentication
```

Comment:
Does this mean that we're sending the `Renku-Auth-Access-Token: {access_token}` to the object store (alongside some `Authorization: Bearer {token}` header for the actual object store access? That would be a big problem as the users access tokens would then suddenly show up in SWITCHengines server logs.

Reply:
Thanks @ableuler for noticing this! I've switched to using git credential helpers and basic auth.